packages/frontend-v2/components/Plan/ScheduleYear.tsx

```diff
@@ -68,7 +60,7 @@ export const ScheduleYear: React.FC<ScheduleYearProps> = ({
   }, 0);
 
   return (
-    <Flex flexDirection="column" height="100%" minHeight="inherit">
```

Comment:
Most of the issue seemed to be a difference with how Safari handles `height:100%` and minHeight, so I went through and replaced that with some flexbox styling.